test/sort.jl

```diff
@@ -1,21 +1,20 @@
 module TestSort
-    using Base.Test
-    using DataTables
+    using Base.Test, DataTables
 
-    dv1 = NullableArray(Nullable{Int}[9, 1, 8, Nullable(), 3, 3, 7, Nullable()])
-    dv2 = NullableArray(Nullable{Int}[9, 1, 8, Nullable(), 3, 3, 7, Nullable()])
-    dv3 = NullableArray(1:8)
-    cv1 = NullableCategoricalArray(dv1, ordered=true)
+    dv1 = [9, 1, 8, null, 3, 3, 7, null]
+    dv2 = [9, 1, 8, null, 3, 3, 7, null]
+    dv3 = Vector{Union{Int, Null}}(1:8)
+    cv1 = CategoricalArray(dv1, ordered=true)
```

Comment:
Should test nullable vector.

Reply:
This one is nullable, since it converts `dv1`